### ISSUE TITLE
Admin initialization - ImproperlyConfigured: AUTH_USER_MODEL refers to model 'auth.User' that has not been installed

### ISSUE THREAD

**Issue by MartNytrm** [haiwen/seahub#797]
Hallo,

I'm at installing Seafile 4.3.3 Pro Version on an Ubuntu server 14.04 64 bit (Python 2.7.6). Everything went fine, except the start of seahub (".../seafile-server-latest/seahub.sh start") and especially the initialization of the admin account, that comes with it. Same happens of course with running "".../seafile-server-latest/reset-admin.sh". I always get the error mesage: "ImproperlyConfigured: AUTH_USER_MODEL refers to model 'auth.User' that has not been installed".

It might have something to do with Django. As I see it (https://groups.google.com/forum/#!topic/seafile/Ao_54GbqV0I), Seafile brings its own Django with it and maybe here is the reason for this issue: On this Ubuntu server, there is Django 1.6.1 installed (via system repos), which is used by another piece of software. And maybe there is friction between these two instances?

Is my assumption appropriate? Or is it something other?
What can be done to solve this issue?

Would be great, if anybody has an idea or knows this problem. Would really appreciate some help, because I'm not familiar with Django.

Much thanks!

Greets

Nytrm

**Comment by mikeymclellan** [comment]
I managed to work around this by 

```cd seafile-server-latest/seahub
ln -s thirdpart/registration .``

and then starting seahub. 

I'm not 100% sure why, but for some reason on my Raspberry Pi it can't find the registration module. I'm guessing some py search path is wrong. The above fixes it for me.

**Comment by MartNytrm** [comment]
Hello mikeymclellan,

thanks for the hint. I will try it the next time I install seafile again. I thought I've tested the symlinks and everything seemed fine, but if this was the answer for me too, it would be pretty simple. We will see next time ;-).

For this time this worked for me:
As I mentioned, I have a later version of Django installed on my server. These Django version was installed out of the repositories of my distro, so Django in this version was/is set as global python site package, which is read by seahub by default. So it always had this problem.
What I did is, I set up a local virtual python environment for seahub with virtualenv without any site packages loaded. Then I installed the python dependencies of seahub like mentioned in the manual (boto, mysql-python, etc.) with pip. Load this environment for seahub, before you start seahub. Done! So Seahub is mostly independent of your system python environment and will run safely.

Greets


Nytrm
